### PR TITLE
docs

### PR DESCRIPTION
**Pull request by philschmid**

### CODE REVIEW

File: docs/source/guides/private_link.mdx [hunk 1]
```diff
@@ -4,32 +4,63 @@ Security and secure inference are key principles of Inference Endpoints. We curr
...
+## 2. Create the Endpoint
+
+After providing your AWS Account ID and any other required information, click Create Endpoint. The endpoint creation process will begin.
+
+<img
+  src="https://raw.githubusercontent.com/huggingface/hf-endpoints-documentation/main/assets/6_2_endpoint_creation.png"
+  alt="creation process"
+/>
+
+After a few minutes, the endpoint will be created and you will see the VPC Service Name in the overview. This name is necessary for creating the VPC Interface Endpoint in your AWS account.
```
Comment (Michellehbn) [inline docs/source/guides/private_link.mdx:25]:
```suggestion
Once the endpoint is deployed, you will see the VPC Service Name in the overview. This name is necessary for creating the VPC Interface Endpoint in your AWS account.
```

File: docs/source/guides/private_link.mdx [hunk 2]
```diff
@@ -4,32 +4,63 @@ Security and secure inference are key principles of Inference Endpoints. We curr
...
 
-An example of the VPC Endpoint configuration is shown below. You will need to select the VPC and subnets, as well as the security groups you want to use.
+<img
+  src="https://raw.githubusercontent.com/huggingface/hf-endpoints-documentation/main/assets/6_5_add_vpc.png"
+  alt="vpc endpoint"
+/>
+
+## 4. Endpoint Running
+
+After the VPC Endpoint status changes from pending to available, you should see an Endpoint URL in the overview. This URL can now be used inside your VPC to access your endpoint in a secure and protected way, ensuring traffic is only occurring between the two endpoints and will never leave AWS.
```
Comment (Michellehbn) [inline docs/source/guides/private_link.mdx:51]:
```suggestion
After the VPC Endpoint status changes from pending to available, you will see an Endpoint URL in the overview. This URL can now be used inside your VPC to access your endpoint privately, ensuring traffic is only occurring between the two endpoints and will never leave AWS.
```

**Review by Michellehbn — APPROVED** [review]
A few minor suggestions ! Thanks so much @philschmid !

**Review by christophe-rannou — APPROVED** [review]

File: docs/source/guides/private_link.mdx (hunk 1 above)
Comment (philschmid) [reply docs/source/guides/private_link.mdx:25]: Thats not correct. Its not deployed.

**Pull request merged**